modules/storage.js

```diff
@@ -0,0 +1,13 @@
+const saveTasks = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
```

Comment:
The arrow function is not properly used here, since it's just a single-line function, you could remove the `{}`
`const saveTasks = (tasks) => localStorage.setItem('tasks', JSON.stringify(tasks));`